lang/fpc/Portfile

```diff
@@ -18,31 +18,41 @@ homepage            http://www.freepascal.org
 master_sites        sourceforge:freepascal
 
 set src             ${name}build-${version}.tar.gz
-# the 3.0.0 bootstrap compiler
-set pp              x86_64-macosx-10.7-ppcx64.tar.bz2
+# the 3.2.0 bootstrap compiler
+set pp              fpc-3.2.0.intel-macosx.dmg
 distfiles           ${src} ${pp}
 checksums           ${src} \
-                    rmd160  f9414fe72342cff4317029f3a83a90e8dc710389 \
-                    sha256  f66514e6f2c2e4e1bccccb4d554c24b77682ed61c87811ae5dd210f421855e76 \
-                    size    68908523 \
+                    rmd160  c8d56acb409a6d5c186b0aaed38dc02dfa3bcb69 \
+                    sha256  f9b914eace989a023fb953da203dc0d973b44487568b4138c7d5b9613d7d6838 \
+                    size    80698915 \
                     ${pp} \
-                    rmd160  25fb3c2eb9fa364ee56a5f3c2667151143f39777 \
-                    sha256  a67ef5def356d122a4692e21b209c328f6d46deef4539f4d4506c3dc1eecb4b0 \
-                    size    1095556
+                    rmd160  bc6be131b47122568dcbd73c9a47c5d8c46cbbb3 \
+                    sha256  77a55f4ab985124bb550e6fa80dfbb2c3ee449c41261325f2623e01a6afd13ad \
+                    size    169412171
 supported_archs     x86_64
 
 # doesn't install shared libraries
 installs_libs       no
 
 extract.only        ${src}
 
+# get the bootstrap compiler from the dmg
+post-extract {
+    system -W ${distpath} "hdiutil attach fpc-3.2.0.intel-macosx.dmg -noverify"
```

Comment:
Please specify a custom `-mountpoint`. The default mountpoint is not reliable.

https://github.com/macports/macports-base/blob/e82a6397e5a404ac157552129015b21557f57d53/src/port1.0/portextract.tcl#L119-L122

Reply:
hey, thanks. Will do. By the way, have you seen any way to use MacPorts' built-in ```use_dmg``` functionality when you have another distfile to decompress as well? I tried several things but it appears I have to do it manually...

Reply:
MacPorts' built-in extract phase does not support multiple formats, so you have to do it manually.

BTW, `-readonly` and `-nobrowse` flags are also recommended, see the man page `hdiutil(1)` for more information.

Reply:
Yep, I can see why. Thanks. Guess. I should have just copied the whole enchilada out of the portextract.tcl file, but I was  distracted by the pkgutil business adding another layer :>